dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerMainIntegrationTest.java

```diff
@@ -117,8 +117,8 @@ public void testArtifacts()
   public void testArtifacts_noError()
       throws IOException, RepositoryException, TransformerException, XMLStreamException,
           LinkageCheckResultException {
-    // gax does not have any linkage errors
-    LinkageCheckerMain.main(new String[] {"-a", "com.google.api:gax:1.56.0"});
```

Comment:
With excluding optional dependencies, we cannot use gax for an artifact for this test case.

Reply:
What optional dependencies does GAX have?

Reply:
It's gax's transitive dependency "commons-logging:commons-logging:1.2". 

```
Class org.apache.avalon.framework.logger.Logger is not found;
  referenced by 1 class file
    org.apache.commons.logging.impl.AvalonLogger (commons-logging:commons-logging:1.2)
Class org.apache.log.Hierarchy is not found;
  referenced by 1 class file
    org.apache.commons.logging.impl.LogKitLogger (commons-logging:commons-logging:1.2)
Class org.apache.log.Logger is not found;
  referenced by 1 class file
    org.apache.commons.logging.impl.LogKitLogger (commons-logging:commons-logging:1.2)

commons-logging:commons-logging:1.2 is at:
  com.google.api:gax:jar:1.56.0 / com.google.auth:google-auth-library-oauth2-http:0.20.0 (compile) / com.google.http-client:google-http-client:1.34.0 (compile) / org.apache.httpcomponents:httpclient:4.5.10 (compile) / commons-logging:commons-logging:1.2 (compile)
```

https://search.maven.org/artifact/commons-logging/commons-logging/1.2/jar shows

```
    <dependency>
      <groupId>logkit</groupId>
      <artifactId>logkit</artifactId>
      <version>1.0.1</version>
      <optional>true</optional>
    </dependency>
    <dependency>
      <groupId>avalon-framework</groupId>
      <artifactId>avalon-framework</artifactId>
      <version>4.1.5</version>
      <optional>true</optional>
    </dependency>
```